src/tdd/isograma.test.ts

```diff
@@ -15,5 +16,15 @@ describe("Testes Isograma", () => {
 
     test("True ou false sem espaços", () => {
         expect(isograma("-   six-year-old")).toBe(true);
+=======
+    test("True ou false sem espaços", () => {
+        expect(isograma("-   background")).toBe(true);
+    });
+    test("True ou false sem espaços", () => {
+        expect(isograma("-   downstream")).toBe(true);
+    });
+    test("True ou false sem espaços", () => {
+        expect(isograma("-   six-year-old")).toBe(false);
+>>>>>>> Stashed changes
```

Comment:
Quando for resolver conflitos durante `git commit`, `git push` ou `git merge` lembra de remover esses textos que aparecem, eles só são colocados pra te ajudar a visualizar as partes que estão conflituosas.  
Ex:
- "<<<<<<< Updated upstream"
- "======="
- ">>>>>>> Stashed changes"